### ISSUE TITLE
Using loss_output_conversion in input query in combination with elastic loss gives faulty results

### ISSUE THREAD

**Issue by kndehaan** [quintel/etengine#1614]
**Issue**
In Koolstofmodellering project, @kaskranenburgQ and I were writing inputs representing the production of processes (in PJ). This is converted to update the demand of the production node, using the efficiency of the node, see simplified example below.

```
UPDATE(
  V(energy_production_fischer_tropsch_synthetic_must_run),
  demand,
  DIVIDE(
    PRODUCT(USER_INPUT(), BILLIONS),
    V(energy_production_fischer_tropsch_synthetic_must_run, "1.0 - loss_output_conversion")
  )
)
```

The in- and outputs of `energy_production_fischer_tropsch_synthetic_must_run` are as follows:
```
- input.electricity = 0.55
- input.hydrogen = 0.45
- output.kerosene = 0.5
- output.loss = elastic
- output.naphtha = 0.25
```

When setting the input in the front-end, it gives very strange results in scenario dashboard items, which is reflected in the primary energy use chart.

Example 40 PJ production:
<img width="2323" height="1254" alt="Image" src="https://github.com/user-attachments/assets/017626c6-b4bb-4d7a-b9ea-dc683e07d5d1" />

Example 200 PJ production:
<img width="2316" height="1258" alt="Image" src="https://github.com/user-attachments/assets/f7c98eeb-495c-4c56-a4f8-b3bbdee298f5" />

From the primary energy use chart, the strange results seem merit order-dependent. Additionally, when there's an electricity shortage, the results seem normal again:

Example 1000 PJ production:
<img width="2326" height="1260" alt="Image" src="https://github.com/user-attachments/assets/063c6ee9-c854-4667-a10d-f12ec88e8999" />

**Issue**
What we know: 
- the "loss_output_conversion" in the input query, in combination with the node having a `output.loss = elastic` causes the issue somewhere (changing elastic to a hard coded loss value gives normal results)
- the issue is related to electricity use of the process node
- using "kerosene_output_conversion + naphtha_output_conversion" instead of "1.0 - loss_output_conversion" circumvents the issue (normal results are obtained). When using both in the GQL sandbox, the same results are given

I already quickly tried to reproduce the issue with the existing input `output_of_energy_production_synthetic_methanol` that sets the demand of `energy_production_synthetic_methanol` and has a similar setup as the new process node mentioned above, but I haven't been able to reproduce the issue there so far. 

**Solution**
It would be good to reproduce the issue and see what exactly causes the fact that the loss doesn't seem to be able to be calculated (e.g. multiple in- and outputs of carriers, specific type of carriers, etc.). Requires both modeller and developer to look in to.
